packages/api-electron/preload.js

```diff
@@ -41,3 +45,26 @@ class ScreenSnippet {
 window.ssf.ScreenSnippet = ScreenSnippet;
 
 window.ssf.app.ready = () => Promise.resolve();
+
+class MessageService {
+  static send(windowId, topic, message) {
+    const fromId = window.ssf.Window.getCurrentWindowId();
```

Comment:
Is this needed here, or can the `fromId` be extracted in the main process from `event.sender`?